packages/container/src/components/FieldPluginContainer.tsx

```diff
@@ -97,21 +102,24 @@ const useSandbox = (
   })
   const [content, setContent] = useState<unknown>(initialContent)
   const [language, setLanguage] = useState<string>('')
+  const [stateChangedCallbackId, setStateChangedCallbackId] = useState<string>()
 
-  const loadedData = useMemo<StateChangedMessage>(
+  const stateChangedData = useMemo<StateChangedMessage>(
     () => ({
       model: content,
       schema: schema,
-      action: 'loaded',
+      action: 'state-changed',
       uid,
       blockId: undefined,
       language,
       spaceId: null,
       story,
       storyId: undefined,
       token: null,
+      isModalOpen,
+      callbackId: stateChangedCallbackId,
     }),
-    [uid, content, language, schema, story],
+    [uid, content, language, schema, story, isModalOpen],
```

Comment:
Hey @eunjae-lee, here eslint is warning for missing dependencies:

![image](https://github.com/storyblok/field-plugin/assets/1240591/fdc8091b-33c9-462c-b5fd-de2435478dfa)